spork/main.py

```diff
@@ -12,7 +12,7 @@
 pygame.mixer.quit() # Hack to stop sound lagging.
 pygame.mixer.init(22050, -16, 2, 1024)
 clock = pygame.time.Clock()
-built_sprites = pygame.sprite.OrderedUpdates()
+built_sprites = pygame.sprite.Group()
```

Comment:
We need to use the `OrderedUpdates` class instead of `Group` so we can guarantee the order of drawn sprites.

```suggestion
built_sprites = pygame.sprite.OrderedUpdates()
```